www/index.html

```diff
@@ -155,87 +168,186 @@
 <script src="scripts/app.js"></script>
 <script type="text/javascript" charset="utf-8">
 
-    document.addEventListener('WebComponentsReady', function () {
+document.addEventListener('WebComponentsReady', function () {
 
-        var button = document.getElementById('barFinder');
+    var button = document.getElementById('barFinder');
 
-        button.addEventListener('click', function () {
-            var spinner    = document.querySelector('paper-spinner');
-            spinner.active = true;
+    button.addEventListener('click', function () {
+        var spinner    = document.querySelector('paper-spinner');
+        spinner.active = true;
 
-            var options = {
-                enableHighAccuracy: true,
-                maximumAge: 0
-            };
+        var options = {
+            enableHighAccuracy: true,
+            maximumAge: 0
+        };
+
+        navigator.geolocation.getCurrentPosition(geolocationSuccess, geolocationError, options);
+    });
+
+    function geolocationError(error) {
+        console.warn('ERROR(' + error.code + '): ' + error.message);
+
+        var toast = document.getElementById('gelocationError');
+        toast.open();
+    }
+
+    function geolocationSuccess(position) {
+
+        var spinner       = document.querySelector('paper-spinner');
+        var gMap          = document.querySelector('google-map');
+        var crd           = position.coords;
+        var refreshButton = document.getElementById('refreshButton');
+        var UserMarker    = document.getElementById('user_marker');
+        var pages         = document.querySelector('iron-pages');
+        var tabs          = document.getElementById('headerTabs');
+        var finderMenu    = document.getElementById('finderMenu');
+        var ajaxElement   = document.getElementById('ajax-bars');
+
+        ajaxElement.url     = "http://api.happlyhour.io/bars";
+        ajaxElement.params  = {"latitude": crd.latitude, "longitude": crd.longitude};
+        ajaxElement.verbose = true;
+        ajaxElement.addEventListener('response', function (request) {
+
+            var aReponseApi = request.detail.response;
```

Comment:
not very bad but instead of prefix your variables with `a`, `o`...., you can write that above :
```js
/** @var array reponseApi **/
```

and your IDE will love you and will understand that 😸